src/Administration/Resources/app/administration/src/app/component/form/sw-text-editor/sw-text-editor-toolbar-button/index.js

```diff
@@ -168,5 +171,11 @@ Component.register('sw-text-editor-toolbar-button', {
             flyoutLinkMenu.style.setProperty('--flyoutLinkLeftOffset', `${flyoutLinkLeftOffset}px`);
             flyoutLinkMenu.style.setProperty('--arrow-position', `${arrowPosition}px`);
         },
+
+        getFlyoutMenuContainerElement(element) {
```

Comment:
Hey there, here is a comment from the internal review process: 

> We don't want to do this manually anymore because there are several things to consider. I would suggest to use the sw-popover component for this.

Could you take a look at this?